src/flag_gems/utils/pointwise_dynamic.py

```diff
@@ -1075,7 +1075,8 @@ def __init__(self, op_desc: FunctionSchema, scalar_fn: JITFunction, config=None)
 
         assert isinstance(scalar_fn, JITFunction)
         self._scalar_fn = scalar_fn
-        self._scalar_fn_cache_key = scalar_fn.cache_key
+        # FIXME: cache_key is too long and make open file failed.
+        self._scalar_fn_cache_key = scalar_fn.cache_key[:33]
```

Comment:
I'm not sure if slicing will bring risk. theoretically, there might exist a small probability that two keys have the same prefix.

Reply:
it risks but very affects very little, the collision probability is $P≈2^{-128}$, if do insist that should be modified, we will fix it. Anyway, we will deep in again and check the root reason of the failure, sorry about that